test/Upgradeable.ts

```diff
@@ -389,11 +392,11 @@ describe("TimeLockPool", function () {
         });
     });
 
-    
-   
 
 
-    
-    
+
+
+
+
```

Comment:
Are all these blank lines here on purpose?

Reply:
Looks like it is the prettier.